frontend/src/views/Abfrage.vue

```diff
@@ -466,40 +485,34 @@ export default class Abfrage extends Mixins(
     this.initializTreeItemsToOpen();
   }
 
-  private saveAbfrageInStore(abfrage: InfrastrukturabfrageModel) {
-    this.$store.commit("search/selectedAbfrage", _.cloneDeep(abfrage));
-  }
-
-  private async abfrageFreigeben(): Promise<void> {
-    if (this.validate()) {
-      if (
-        this.abfrageWrapped.infrastrukturabfrage.abfrage.statusAbfrage ===
-        AbfrageListElementDtoStatusAbfrageEnum.Angelegt
-      ) {
-        this.infrastrukturabfrageFreigeben();
-      } else {
-        this.showWarningInInformationList('Die Abfrage muss den Status "angelegt" besitzen');
-      }
-    } else {
-      this.showWarningInInformationList("Es gibt noch Validierungsfehler");
-    }
-  }
-
-  private async infrastrukturabfrageFreigeben(): Promise<void> {
+  private async startStatusUebergang(transition: TransitionDto) {
     const validationMessage: string | null = this.findFaultInInfrastrukturabfrageForSave(
       this.abfrageWrapped.infrastrukturabfrage
     );
     if (_.isNil(validationMessage)) {
       await this.updateInfrastrukturabfrage(this.abfrageWrapped.infrastrukturabfrage, true);
-      this.freigabInfrastrukturabfrage(this.abfrageWrapped.infrastrukturabfrage.id as string, true).then(() => {
-        this.returnToUebersicht("Die Abfrage wurde erfolgreich freigegeben", Levels.SUCCESS);
-      });
-      this.initializeFormulare();
+      const requestSuccessful = await this.statusUebergangRequest(transition, this.abfrageId);
+      if (requestSuccessful) {
+        if (!(transition.buttonName === "IN BEARBEITUNG SETZEN")) {
+          this.returnToUebersicht("Die Abfrage hatte einen erfolgreichen Statuswechsel", Levels.SUCCESS);
+        } else {
+          this.getAbfrageById();
```

Comment:
Die Methode `getAbfrageById` umbennnen. Da keine Abfrage zurückgegeben wird.

Die Methode setzt die im Parameter ID referenzierte Abfrage in den Store. -> Evtl. `setSelectedAbfrageInStore` oder ähnlich.